### DISCUSSION TITLE
sampling error

### DISCUSSION THREAD

**Discussion by karabekmez** [SysBioChalmers/yeast-GEM#348] [HELP!]
Hi,
I'm trying to use sampling functions in cobratoolbox. When I use Yeast v8.7.0 or Yeast v7.6.0 with ACHR sampling I got an error after reducing model, the reduced model fail to pass consistency test: 
"Error using solveCobraLP
[gurobi] Primal optimality condition in solveCobraLP not satisfied, residual = 0.0001359, while feasTol = 1e-06"

However, I dont have this error if I use Yeast 6.0.

Additionally, if I use CHRR sampling I have an error for all three models that: 
"Error using mve_solver_cobra
x0 not interior"

Is there any simple modification I should do to avoid these errors or should I just use the working combination (v6.0 + ACHR). I want to use as up-to-date model as possible.

Thanks.

**Comment by edkerk** [comment]
Have you tried setting different constraints for exchange (and possible some intracellular) reactions? And have you tried using different solvers, I assume you used Gurobi?
